doc/adr/0002-global-and-state-channel-snapshots.md

```diff
@@ -25,7 +25,7 @@ will be aggregated by Layer 0 and proposed to the Global snapshot.
 
 ### Global snapshots
 
-Global snapshots are being created on Layer 0. They contains both DAG blocks and
+Global snapshots are being created on Layer 0. They may contain both DAG blocks and
```

Comment:
@kpudlik if they "may contain" it means they can be completely empty? I'd specify that in the ADR

Reply:
Yes, time-based trigger won't put DAG blocks there.